src/components/Header.js

```diff
@@ -1,7 +1,22 @@
+import Link from 'next/link';
+import styled from 'styled-components';
+
+const Container = styled.div`
+	display: flex;
+	flex-wrap: wrap;
+	gap: 16px;
+`;
+
 export default function Header() {
 	return (
-		<header>
-			<h1>Movie Bookmark App</h1>
-		</header>
+		<Container>
+			<header>
+				{/* <h1>Movie Bookmark App</h1> */}
```

Comment:
Please remove unused and commented out code.

Reply:
deleted